during_the_workshop.md

```diff
@@ -75,34 +75,99 @@ Now we know how to feed data into Chimera we need to work out how to do this mor
 We want to use [USGS](https://earthquake.usgs.gov/earthquakes/feed/v1.0/geojson.php) as our source. They have various regularly updated feeds. Start off using the [hourly feed](https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_hour.geojson). We need to write a script to convert this data to the pipe delimited format that `cliapp` can read.
```

Comment:
Just a thought; the instruction to create a script is a little back to front. Should we perhaps encourage trying things over the terminal first and then ask to create a script at the end of the section? (it's a shorter feedback loop and some learners fall back to this anyway and forget they need to create a script before starting the automation section).